content/help/addons/rich-editor.md

```diff
@@ -0,0 +1,156 @@
+
+## Rich Embeds
+
+Rich Editor supports embedding of external content within posts. Use the embed menu to transform a link into a rich content embed.
+
+![Embed menu](/img/help/addons/rich-editor/embed-menu.png)
+
+While we fetch the data for the embedded content, the link appears as a link with a loading indicator.
+
+![Embed loading indicator](/img/help/addons/rich-editor/embed-loading.png)
+
+Once the embed is ready it will display in it's final form directly in the editor.
+
+### Quotes
+
+If you embed a link from the same forum, or click on the "Quote" button, and embedded version of that content will be embedded in your post.
+
+If the content is too long, an expand and collapse menu will be shown for that Rich Quote.
+
+![Internal Link Quoted content](/img/help/addons/rich-editor/embed-quote.png)
+
+### Links
+
+Rich Editor offers a beautiful link embed. It fetches titles, descriptions, and images from the linked URL and displays them in a sleek modern view.
+
+![External Link Embedded content](/img/help/addons/rich-editor/embed-site.png)
+
+### Twitter
+
+Twitter links get transformed into embedded tweets.
+
+![Twitter Embed](/img/help/addons/rich-editor/embed-twitter.png)
+
+### Videos
+
+Videos appear as a fast, responsive thumbnail, with the full video loading on when the play button is clicked. Rich Editor currently supports:
+
+-   Youtube
+-   Vimeo
+-   Twitch
+-   Wistia
+
+![Video Embed](/img/help/addons/rich-editor/embed-video.png)
+
+### SoundCloud
+
+![Soundcloud Embed](/img/help/addons/rich-editor/embed-soundcloud.png)
+
+### Other Embeds
+
+In additional to these types of embedded content, Rich Editor currently supports:
+
+-   Giphy
+-   Instagram
+-   Getty Images
+-   Codepen
+
+## Enabling Rich Editor
+
+Rich Editor co-exists alongside existing posting formats such as "Markdown", "BBCode", and "HTML" from [Advanced Editor](/help/addons/advanced-editor).
+
+If you are an administrator you can use the Rich Editor format by:
+
+1. Navigate to the Addons page.
+2. Toggle the Rich Editor (WIP) plugin.
+
+Enabling Rich Editor will automatically make "Rich" your default post format. To change your format back, go to the Posting page in dashboard settings, or navigate directly to `/vanilla/settings/posting`.
+
+![Posting Format Settings page Rich](/img/help/addons/rich-editor/post-format-rich.png)
+
+## FAQ
+
+### What do I do with my existing Advanced Editor posts?
+
+Rich Editor can function side by side with Advanced Editor. Leave Advanced Editor enabled in addition to Rich Editor, and older posts will still be editable using Advanced Editor.
+
+### What causes the warning icon while inserting a Rich Embed?
+
+Some users using browsers with strict privacy settings or adblockers may not be able to load a Rich Embed from certain sites in their browser. The warning icon indicates that all the data necessary to render the embed is present in the editor, but the user's browser is not allowing thet embed to be created.
```

Comment:
Add screen shot

Reply:
I would except for this issue https://github.com/vanilla/vanilla/issues/7606

We can circle back to this screenshot when that is fixed. We should still merge this now so Adrian can look through it and clients can start seeing it.